app/services/payment_providers/stripe/payments/create_service.rb

```diff
@@ -139,6 +171,7 @@ def success_redirect_url
         # to permit 3D secure authentication
         # https://docs.stripe.com/india-recurring-payments
         def off_session?
+          return false if provider_customer.provider_payment_methods == ["customer_balance"]
           invoice.customer.country != "IN"
```

Comment:
Nitpick but for consistency I'd prefer if it looked like `off_session?` in StripeService

But, now I'm wondering why I didn't update this with crypto 🤔 

https://github.com/getlago/lago-api/blob/aba70bdb294c9fcd6ccbc36f9ded1b4155f2b5ba/app/services/invoices/payments/stripe_service.rb#L212-L217